services/ui/ws/window_tree_unittest.cc

```diff
@@ -1325,7 +1325,7 @@ TEST_F(WindowTreeTest, MoveLoopAckOKByWM) {
       ->PerformWindowMove(
           change_id,
           child_tree->ClientWindowIdToTransportId(embed_window_id2_in_child),
-          mojom::MoveLoopSource::MOUSE, gfx::Point(0, 0));
+          mojom::MoveLoopSource::MOUSE, gfx::Point(0, 0), gfx::Vector2d());
```

Comment:
suggestion: maybe instead of adding the mechanical default last parameter value here, we could declare the method with a default value?

then the call sites would not need change. does it make sense?

Reply:
leaving as it is because defaulting requires too much to do still.